CMakeLists.txt

```diff
@@ -1,14 +1,40 @@
 cmake_minimum_required(VERSION 3.4)
-find_package(PythonInterp)
-enable_language(ASM CXX)
-project(v8)
+set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
```

Comment:
I'm okay with this, just wondering if there's a strong reason to disallow them. My goal for this project is that people can just vendor it so it shouldn't be _too_ opinionated.